.changeset/red-lamps-arrive.md

```diff
@@ -0,0 +1,25 @@
+---
+"@lynx-js/react": patch
+---
+
+Support rstest for testing library, you can use rstest with RLTL now:
+
+```ts
+import { defineConfig } from '@rstest/core';
+import { pluginReactLynx } from '@lynx-js/react-rsbuild-plugin';
+import { rstestTestingLibraryPlugin } from '@lynx-js/react/testing-library/plugins';
+
+export default defineConfig({
+  plugins: [
+    rstestTestingLibraryPlugin(),
+    pluginReactLynx({
+      enableTestingLibrary: true,
+    }),
```

Comment:
Instead of a new option, we may use the [`api.context.callerName`](https://rsbuild.rs/api/javascript-api/instance#contextcallername) to see if the plugin is applied by Rstest

```suggestion
    pluginReactLynx(),
```